### DISCUSSION TITLE
Welcome to k8s Discussions!

### DISCUSSION THREAD

**Discussion by jmordica** [voxoco/k8s#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BrunoDuarte** [comment]
Hi, is the asterisk image (and its Dockerfile) restricted to your company? Or, is it open to anyone to download? Just like the other images referenced in YAML. 

By the way, congratulations on this great job, explaining all the details of this infrastructure. 

Regards, 
Bruno Duarte

**Reply by jmordica** [reply]
Thanks for stopping by. It has definitely evolved even since Astricon 2021 and has been refactored to be more generalized.

Our repos are currently private but we are considering open sourcing our whole platform. This config repo was kind of a start to see if there was/is interest in what we have built. As things progress you will see more generalized tooling being made open source and possibly everything eventually.

We don’t mind sharing if you are interested in certain services/code for now. Just reach out and let us know.

Thanks!

**Reply by BrunoDuarte** [reply]
Awesome!

Your presentation was great. I'm planning to build something upon a similar infrastructure, but when I saw your video, you explained a lot of things that I was already looking for. You have synthesized a lot of really useful information over this kind of infrastructure, that is hard to find on the internet. 

I have cloned your repo, and I'm studying it to see how things are being implemented because I started my project on AWS. But I'm still deciding if it will stay on AWS or if I move it to Google Cloud because you convinced me about its advantages. 

I'm not sure if all these things are sensitive information for you, but I really appreciate anything you can share with me.

Thank you.

Regards, 
Bruno Duarte
Email: fbruno.c.duarte@gmail.com
